packages/neuron-ui/src/components/WalletWizard/index.tsx

```diff
@@ -7,10 +7,14 @@ import ImportWallet from './importWallet'
 import CreateWallet from './createWallet'
 
 const Wizard = styled.div`
+  .full-screen & {
+    background-color: white;
+    width: 100vw;
```

Comment:
better to use `width: 100%`, `width: 100vw` will overflow when scroll bar exists.

Reply:
yes, it would have scroll problem. it is not under <MainContent>. I could use 100% for width and height. I would change it